sphinx-docs/usermanual/userman_main.rst

```diff
@@ -0,0 +1,47 @@
+User Manual
+============
+
+If you know which user manual you'd like to see, click to jump to a specific  manual:
+
+* :doc:`userman_admin`
```

Comment:
Consider putting these into subdirectories, especially if you take my suggestion of splitting the specified files into subtopics.